tnet/listener.go

```diff
@@ -41,8 +41,10 @@ func Wrap(l net.Listener) net.Listener {
 // The following can happen:
 // Goroutine 1 is running Accept, and is blocked, waiting for epoll
 // Goroutine 2 calls Close. It sees an extra reference, and so cannot destroy
-//  the socket, but instead decrements a reference, marks the connection as closed
-//  and unblocks epoll.
+//
+//	the socket, but instead decrements a reference, marks the connection as closed
+//	and unblocks epoll.
+//
```

Comment:
should this be part of the previous line versus a preformatted block?

Reply:
good catch. fixed.